satpy/tests/reader_tests/test_seviri_l1b_calibration.py

```diff
@@ -93,108 +98,214 @@
                                [33.934814, 35.33813,  37.251755, 38.655075,
                                 40.56869]], dtype=np.float32)
 
-# --
-
-CAL_DTYPE = np.array([[(0.0208876,  -1.06526761), (0.0278805,  -1.42190546),
-                       (0.0235881,  -1.20299312), (0.00365867,  -0.18659201),
-                       (0.00831811,  -0.42422367), (0.03862197,  -1.96972038),
-                       (0.12674432,  -6.46396025), (0.10396091,  -5.30200645),
-                       (0.20503568, -10.45681949), (0.22231115, -11.33786848),
-                       (0.1576069,  -8.03795174), (0.0373969,  -1.90724192)]],
-                     dtype=[('CalSlope', '>f8'), ('CalOffset', '>f8')])
-
-IR_108_RADIANCES = np.ma.array([[133.06815651,  133.68326355,  134.29837059,  134.91347763,
-                                 135.52858467],
-                                [136.14369171,  136.75879875,  137.37390579,  137.98901283,
-                                 138.60411987],
-                                [139.21922691,  139.83433395,  140.44944099,  141.06454803,
-                                 141.67965507]],
-                               mask=False, dtype=np.float64)
-
-VIS006_RADIANCES = np.ma.array([[13.55605239,  13.61871519,  13.68137799,  13.74404079,
-                                 13.80670359],
-                                [13.86936639,  13.93202919,  13.99469199,  14.05735479,
-                                 14.12001759],
-                                [14.18268039,  14.24534319,  14.30800599,  14.37066879,
-                                 14.43333159]], mask=False, dtype=np.float64)
-
-VIS006_REFLECTANCES = np.array([[65.00454035,  65.30502359,  65.60550682,  65.90599006,
-                                 66.2064733],
-                                [66.50695654,  66.80743977,  67.10792301,  67.40840625,
-                                 67.70888949],
-                                [68.00937272,  68.30985596,  68.6103392,  68.91082244,
-                                 69.21130567]], dtype=np.float64)
-
-IR_108_TBS = np.array([[311.77913132,  312.11070275,  312.44143083,  312.77132215,
-                        313.10038322],
-                       [313.42862046,  313.75604023,  314.0826488,  314.40845236,
-                        314.73345704],
-                       [315.05766888,  315.38109386,  315.70373788,  316.02560677,
-                        316.34670629]], dtype=np.float64)
 
-
-# Calibration type = Effective radiances
-CALIBRATION_TYPE = np.array(
-    [[2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2]], dtype=np.uint8)
-
-
-# This should preferably be put in a helper-module
-# Fixme!
-def assertNumpyArraysEqual(self, other):
-    if self.shape != other.shape:
-        raise AssertionError("Shapes don't match")
-    if not np.allclose(self, other):
-        raise AssertionError("Elements don't match!")
```

Comment:
Replaced with `np.testing.assert_allclose`, which also displays a nice error message if the shapes don't match